workflows/ncn/worker/worker.drain.yaml

```diff
@@ -63,7 +63,7 @@ tasks:
             done
   - name: update-bss
     templateRef:
-      name: iuf-base-template
+      name: ssh-template
```

Comment:
can we just use `kubectl-and-curl-template`? i am guessing the problem here is that iuf-base-template no long mount /host_usr_bin ? i think `kubectl-and-curl-template` still does so we can just use that (it also exports TOKEN automatically)

Reply:
I don't remember the kubectl and curl template being able to run 'cray' commands which I think is why I didn't use it initially. Is it able to run 'cray' 
commands?

Reply:
i see. that's the reason why Atif removed that usr_host_bin lol